docs/_static/llama-stack-spec.html

```diff
@@ -3096,7 +3096,7 @@
             "post": {
                 "responses": {
                     "200": {
-                        "description": "Response from an OpenAI-compatible chat completion request. **OR** Chunk from a streaming response to an OpenAI-compatible chat completion request.",
```

Comment:
Ah there's some bug in the generator that makes it not handle native `|` unions like it does for `typing.Union` for descriptions...

Reply:
Updated `llama_stack/strong_typing/docstring.py` to extract docstring for a AsyncIterator argument from the wrapped type.